invenio_app_ils/records/views.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio App ILS Records views."""
+
+from __future__ import absolute_import, print_function
+
+from flask import Blueprint, request
+from invenio_db import db
+from invenio_records_rest.utils import obj_or_import_string
+from invenio_records_rest.views import pass_record
+
+from invenio_app_ils.circulation.views import IlsResource
+from invenio_app_ils.errors import RelatedRecordError
+from invenio_app_ils.records.api import IlsRecordWithRelations
+
+
+def create_relations_blueprint(app):
+    """Add relations views to the blueprint."""
+    blueprint = Blueprint(
+        "invenio_app_ils_relations",
+        __name__,
+        url_prefix="",
+    )
+
+    endpoints = app.config.get("RECORDS_REST_ENDPOINTS", [])
+    pid_types = ("docid", "serid")
+
+    for pid_type in pid_types:
+        options = endpoints.get(pid_type, {})
+        rec_serializers = {
+            "application/json": (
+                "invenio_records_rest.serializers" ":json_v1_response"
+            )
+        }
+        serializers = {
+            mime: obj_or_import_string(func)
+            for mime, func in rec_serializers.items()
+        }
+        record_relations = RelationResource.as_view(
+            RelationResource.view_name.format(pid_type),
+            serializers=serializers,
+            ctx=dict()
+        )
+        blueprint.add_url_rule(
+            "{0}/relations".format(options["item_route"]),
+            view_func=record_relations,
+            methods=["GET", "POST"]
+        )
+    return blueprint
+
+
+class RelationResource(IlsResource):
+    """Record relation resource."""
+
+    view_name = "{0}_relations"
+
+    @pass_record
+    def get(self, record, **kwargs):
+        """Get related records."""
+        for obj in record["related_records"]:
+            related = IlsRecordWithRelations.get_record_by_pid(
+                obj["pid"],
+                pid_type=obj["pid_type"]
+            )
+            obj["title"] = related.get("title", "")
+            obj["edition"] = related.get("edition", "")
+            obj["language"] = related.get("language", "")
+        pid = record[record.pid_field]
+        for key in list(record.keys()):
+            if key != "related_records":
+                del record[key]
+        return self.make_response(pid, record, 202)
```

Comment:
```suggestion
        return self.make_response(pid, record, 200)
```
any particular need for 202? Same below